src/libraries/System.Private.CoreLib/src/System/IO/Directory.cs

```diff
@@ -284,5 +285,38 @@ public static FileSystemInfo CreateSymbolicLink(string path, string pathToTarget
             FileSystem.VerifyValidPath(linkPath, nameof(linkPath));
             return FileSystem.ResolveLinkTarget(linkPath, returnFinalTarget, isDirectory: true);
         }
+
+        /// <summary>
+        /// Copies a directory into another directory.
+        /// </summary>
+        /// <param name="sourcePath">The source path</param>
+        /// <param name="destinationPath">The destination path</param>
+        /// <param name="recursive"><see langword="true" /> to copy directories, subdirectories, and files in <paramref name="sourcePath"/>; otherwise, <see langword="false" />.</param>
+        /// <returns><see langword="true"/> if the operation succeeded in copying all the specified files and directories; otherwise, <see langword="false"/>.</returns>
+        /// <exception cref="IOException">A write operation fails</exception>
+        public static bool Copy(string sourcePath, string destinationPath, bool recursive) =>
+            Copy(sourcePath, destinationPath, recursive, true);
```

Comment:
We can't do that. The one Copy overload hides the other.
So if we remove the explicit true, it'll will recurse infinite until StackOverflowException. We cannot remove the default value so easily either, because it is so proposed.

See VS Diagnostics
![before](https://user-images.githubusercontent.com/10373452/158450009-c449e7df-b136-4c90-a0b1-195a763b3afa.png)
![after](https://user-images.githubusercontent.com/10373452/158450068-6f373f8e-5be7-4090-bdbc-12d6647108f4.png)

Reply:
Ah, you are right; thanks for clarifying.